__tests__/app.test.js

```diff
@@ -273,7 +273,121 @@ describe("app", () => {
         .send(newComment)
         .expect(400)
         .then(({ body }) => {
-          expect(body.msg).toBe("Bad request: invalid input");
+          expect(body.msg).toBe("Bad request");
+        });
+    });
+  });
+  describe.only("PATCH: /api/reviews/:review_id", () => {
+    test("responds with review object with the correct properties", () => {
+      const update = { inc_votes: 50 };
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(update)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body).toHaveProperty("review_id", expect.any(Number));
+          expect(body).toHaveProperty("title", expect.any(String));
+          expect(body).toHaveProperty("review_body", expect.any(String));
+          expect(body).toHaveProperty("designer", expect.any(String));
+          expect(body).toHaveProperty("review_img_url", expect.any(String));
+          expect(body).toHaveProperty("votes", expect.any(Number));
+          expect(body).toHaveProperty("category", expect.any(String));
+          expect(body).toHaveProperty("owner", expect.any(String));
+          expect(body).toHaveProperty("created_at", expect.any(String));
+        });
+    });
+    test("responds with review with updated values - add", () => {
+      const update = { inc_votes: 50 };
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(update)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            review_id: 1,
+            title: "Agricola",
+            review_body: "Farmyard fun!",
+            designer: "Uwe Rosenberg",
+            review_img_url:
+              "https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700",
+            votes: 51,
+            category: "euro game",
+            owner: "mallionaire",
+            created_at: `2021-01-18T10:00:20.514Z`,
+          });
```

Comment:
A good test, could be made slightly more flexible by using objectContaining or toHaveProperty with the votes defined as what you expect them to be, that way if you ever added another property to the returned object, this test wouldn't need to be changed